### PR TITLE
[DEVHAS-208] Update sli dashboard

### PR DESCRIPTION
**Pull request by kim-tsao**
Signed-off-by: Kim Tsao <ktsao@redhat.com>

### What does this PR do?:

- Updates sli dashboard to use template source

- Ensure hash is updated when there are changes to the dashboard files  

### Which issue(s)/story(ies) does this PR fixes:

https://issues.redhat.com/browse/DEVHAS-208

### PR acceptance criteria:

- [ ] Unit/Functional tests

  

- [ ] Documentation 

   

- [ ] Client Impact

### CODE REVIEW

**Review by johnmcollier — APPROVED** [review]

File: .tekton/push.yaml [hunk 1]
```diff
@@ -21,6 +21,7 @@ spec:
     - name: infra-deployment-update-script
       value: |
         sed -i -e 's|\(https://github.com/redhat-appstudio/application-service/.*?ref=\)\(.*\)|\1{{ revision }}|' -e 's/\(newTag: \).*/\1{{ revision }}/' components/has/kustomization.yaml
+        sed -i -e 's|\(https://github.com/redhat-appstudio/application-service/config/monitoring/?ref=\)\(.*\)|\1{{ revision }}|' -e 's/\(newTag: \).*/\1{{ revision }}/' components/monitoring/grafana/base/kustomization.yaml
```
Comment (johnmcollier) [inline .tekton/push.yaml:24]: Ah, wait I misread this.

Shouldn't this go to the HAS specific folder under https://github.com/redhat-appstudio/infra-deployments/tree/main/components/monitoring/grafana/base?

**Review by johnmcollier — CHANGES_REQUESTED** [review]

File: .tekton/push.yaml (hunk 1 above)
Comment (kim-tsao) [reply .tekton/push.yaml:24]: sigh, looks like they refactored 3 days ago.  All components used to write into the kustomization file according to the instructions

**Review by johnmcollier — APPROVED** [review]

**Pull request merged**